app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide(:title, "About") %>
```

Comment:
tìm hiểu về I18n và sử dụng cho toàn bộ file html nhé

Reply:
Dạ em sẽ tìm hiểu và áp dụng khi học tiếp ạ!

Reply:
tìm hiểu rồi fix luôn đi e
Check lại code xem chỗ nào đang có những text như thế này thì đưa vô i18n hết nhé